samples/gitchurn/src/gitChurnMain/kotlin/GitRevwalk.kt

```diff
@@ -0,0 +1,52 @@
+package sample.gitchurn
+
+
+import kotlinx.cinterop.*
+import libgit2.*
+import platform.posix.alloca
+
+/**
+ * this class provides revwalk functionality without need of memory management etc
```

Comment:
The comment should be a sentence, i.e. start with capital letter and end with a dot.